pkg/hintrunner/zero/zerohint_keccak_test.go

```diff
@@ -0,0 +1,195 @@
+				},
+				check: consecutiveVarAddrResolvedValueEquals(
+					"inputs",
+					[]*fp.Element{
+						feltUint64(18446744073709551615),
+						feltUint64(0),
+						feltUint64(18446744073709551615),
+						feltUint64(0),
+					}),
+			},
+			{
+				operanders: []*hintOperander{
+					{Name: "inputs", Kind: apRelative, Value: addr(7)},
+					{Name: "low", Kind: fpRelative, Value: feltString("18446744073709551616")},
+					{Name: "high", Kind: fpRelative, Value: feltString("18446744073709551616")},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newKeccakWriteArgsHint(ctx.operanders["inputs"], ctx.operanders["low"], ctx.operanders["high"])
+				},
+				check: consecutiveVarAddrResolvedValueEquals(
+					"inputs",
+					[]*fp.Element{
+						feltUint64(0),
+						feltUint64(1),
+						feltUint64(0),
+						feltUint64(1),
+					}),
+			},
+			{
+				operanders: []*hintOperander{
+					{Name: "inputs", Kind: apRelative, Value: addr(7)},
+					{Name: "low", Kind: fpRelative, Value: feltString("18446744073709551617")},
+					{Name: "high", Kind: fpRelative, Value: feltString("18446744073709551617")},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newKeccakWriteArgsHint(ctx.operanders["inputs"], ctx.operanders["low"], ctx.operanders["high"])
+				},
+				check: consecutiveVarAddrResolvedValueEquals(
+					"inputs",
+					[]*fp.Element{
+						feltUint64(1),
+						feltUint64(1),
+						feltUint64(1),
+						feltUint64(1),
+					}),
+			},
+			{
+				operanders: []*hintOperander{
+					{Name: "inputs", Kind: apRelative, Value: addr(7)},
+					{Name: "low", Kind: fpRelative, Value: feltString("18446744073709551617")},
+					{Name: "high", Kind: fpRelative, Value: feltString("18446744073709551617")},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newKeccakWriteArgsHint(ctx.operanders["inputs"], ctx.operanders["low"], ctx.operanders["high"])
+				},
+				check: consecutiveVarAddrResolvedValueEquals(
+					"inputs",
+					[]*fp.Element{
+						feltUint64(1),
+						feltUint64(1),
+						feltUint64(1),
+						feltUint64(1),
+					}),
+			},
+			{
+				operanders: []*hintOperander{
+					{Name: "inputs", Kind: apRelative, Value: addr(7)},
+					{Name: "low", Kind: fpRelative, Value: feltString("340282366920938463463374607431768211455")},
+					{Name: "high", Kind: fpRelative, Value: feltString("340282366920938463463374607431768211455")},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newKeccakWriteArgsHint(ctx.operanders["inputs"], ctx.operanders["low"], ctx.operanders["high"])
+				},
+				check: consecutiveVarAddrResolvedValueEquals(
+					"inputs",
+					[]*fp.Element{
+						feltUint64(18446744073709551615),
+						feltUint64(18446744073709551615),
+						feltUint64(18446744073709551615),
+						feltUint64(18446744073709551615),
+					}),
+			},
+		},
+	})
+}
```

Comment:
You could add one more test, where all the result values are different than 0 and 1

Reply:
done!